login/views.py

```diff
@@ -103,13 +104,17 @@ def verify_otp(request):
                 userObj = authenticate(username=request.session['username'], password=request.session['password'])
                 login(request,userObj)
                 request.session['last_activity'] = datetime.now().strftime("%m/%d/%Y, %H:%M:%S")
-                return HttpResponseRedirect('http://127.0.0.1:8000/user_home')
+
+                if request.user.Profile_User.privilege_id.user_type == settings.SB_USER_TYPE_CUSTOMER:
+                    return HttpResponseRedirect('http://127.0.0.1:8000/user_home')
```

Comment:
Like Akshay mentioned earlier, it's better to use relative path instead of direct url.

For instance, relative path for this would be just,
return HttpResponseRedirect('/user_home/')

Reply:
Also feel free to overwrite your changes to this file over mine.